internal/a3m/config.go

```diff
@@ -1,6 +1,28 @@
 package a3m
 
+import transferservicev1beta1 "go.buf.build/grpc/go/artefactual/a3m/a3m/api/transferservice/v1beta1"
+
 type Config struct {
-	Address  string
+	Address string
+	Name    string
+	Processing
 	ShareDir string
 }
+
+type Processing struct {
+	AssignUuidsToDirectories                     bool
+	ExamineContents                              bool
+	GenerateTransferStructureReport              bool
+	DocumentEmptyDirectories                     bool
+	ExtractPackages                              bool
+	DeletePackagesAfterExtraction                bool
+	IdentifyTransfer                             bool
+	IdentifySubmissionAndMetadata                bool
+	IdentifyBeforeNormalization                  bool
+	Normalize                                    bool
+	TranscribeFiles                              bool
+	PerformPolicyChecksOnOriginals               bool
+	PerformPolicyChecksOnPreservationDerivatives bool
+	AipCompressionLevel                          int
+	AipCompressionAlgorithm                      transferservicev1beta1.ProcessingConfig_AIPCompressionAlgorithm
+}
```

Comment:
It would be nice to add a comment in Processing to clarify the source of all these fields, e.g.:

    // Processing mirrors all the processing configuration fields in transferservicev1beta1.ProcessingConfig.
    // The goal is to reflect all the fields available in the upstream API.

Viper provides its own extensible [unmarshal](https://pkg.go.dev/github.com/spf13/viper#readme-unmarshaling) mechanism, maybe something we could explore in the future.